source/_patterns/02-molecules/hero/hero.twig

```diff
@@ -78,6 +79,11 @@
               <a {{ hero.aside.link.is_external ? 'class="ext"' -}} href="{{ hero.aside.link.url }}">{{ hero.aside.link.text }}</a>
             </div>
           {% endif %}
+
```

Comment:
Try using a card component here.

Reply:
@ivanbueno-jcc  will use the card component.

Reply:
Will delete the /moelcules/location-details folder

Reply:
@ivanbueno-jcc  deleted the unused files.